### PR TITLE
update INSTALL

### PR DESCRIPTION
**Pull request by bjlittle**
Refresh the `INSTALL` instructions.

### CODE REVIEW

**Comment by bjlittle** [comment]
Closes https://github.com/SciTools/cf-units/issues/117

**Comment by bjlittle** [comment]
Happy to rebase once https://github.com/SciTools/cf-units/pull/123 is merged...

File: INSTALL [hunk 1]
```diff
@@ -9,27 +9,27 @@ cf-units is available using conda for the following platforms:
...
+for example from https://conda.io/miniconda.html.
 
 Once conda is installed, you can install cf-units using conda on any platform with
 the following command::
 
   conda install -c conda-forge cf-units
 
 Further documentation on using conda and the features it provides can be found
-at http://conda.pydata.org/docs/intro.html.
+at http://conda.io/docs/.
```
Comment (DPeterK) [inline INSTALL]: `http` --> `https`

File: INSTALL [hunk 2]
```diff
@@ -55,34 +55,40 @@ If you are installing dependencies with a package manager on Linux,
 you may need to install the development packages (look for a "-dev"
 postfix) in addition to the core packages.
 
-python 2.7 or later (http://www.python.org/)
-    cf-units requires Python 2.7 or later, but is not currently compatible with
-    Python 3.
+python 2.7 or later (https://www.python.org/)
+    cf-units requires Python 2.7 or later.
```
Comment (DPeterK) [inline INSTALL]: This line feels tautological now – it's not saying anything that hasn't already been said on the line above. If we wanted to be explicit and say something like `cf-units is compatible with Python 2 and Python 3`, but, really, that's also a given...

**Review by DPeterK — COMMENTED** [review]
@bjlittle I think there are a couple of bits that could do with a little tidy. Otherwise it's a nice update 👍

**Review by pelson — APPROVED** [review]
@dkillick has this one under control 👍

**Comment by bjlittle** [comment]
@dkillick Rebased and addressed your review actions. As agreed, running with this as is, thanks!

**Pull request merged**